src/components/RestaurantHeader/RestaurantHeader.tsx

```diff
@@ -3,29 +3,31 @@ import type { Restaurant } from "../../services/restaurant"
 import { ImageBackground, StyleSheet, View, Text } from "react-native"
 import Typography from "../Typography"
 
-const RestaurantHeader: React.FC<{ restaurant: Restaurant }> = ({
-  restaurant,
-}) => {
+type Props = {
+  restaurant: Restaurant | null
+}
+
+const RestaurantHeader: React.FC<Props> = ({ restaurant }) => {
   return (
     <View style={styles.restaurantHeader}>
-      <ImageBackground source={{ uri: `url(/${restaurant.images.banner})` }}>
+      <ImageBackground source={{ uri: `${restaurant?.images.banner}` }}>
```

Comment:
Is this working? I don’t see it in the screenshots.

Reply:
Thank again for taking the time to help with sorting this out.